src/components/ContinuationIn/ContinuationAuthorization.vue

```diff
@@ -292,15 +292,17 @@ export default class ExtraproRegistration extends Mixins(DocumentMixin) {
   }
 
   @Watch('authorization', { deep: true })
+  @Watch('minAuthorizationDate') // re-validate when the min Authorization Date changes
   @Emit('valid')
   private onComponentValid (): boolean {
     // sync local object to the store
     this.setContinuationAuthorization(this.authorization)
 
-    // this component is valid if we have the authorization date
+    // this component is valid if we have the valid authorization date
     // and at least one file uploaded
     return (
       !!this.authorization.date &&
+      new Date(this.minAuthorizationDate) <= new Date(this.authorization.date) &&
```

Comment:
This is a duplicate of the rules on lines 196-199.

My idea was to use a v-model variable on the v-form. I think onGetShowErrors() will automatically run when minAuthorizationDate changes, which will call the validation functions. Then, here, you can just check if the form is valid.

Reply:
> This is a duplicate of the rules on lines 196-199.
> 
> My idea was to use a v-model variable on the v-form. I think onGetShowErrors() will automatically run when minAuthorizationDate changes, which will call the validation functions. Then, here, you can just check if the form is valid.

The form will declare itself as valid even when the authorizationDate is invalid. It seems the formRef.validate() will be valid even when all fields are empty - is it because all sub custom components don't support form validation?

Reply:
Oh, you're on to something.

The DatePicker is one of our custom components, so no, it doesn't support form validation here... but internally it does (since it uses its own form). 

On lines 289 and 290, you call a public method in that component. There is also `isDateValid()` that you might be able to call to check if it's valid.

Reply:
https://github.com/bcgov/bcrs-shared-components/blob/83cfe9c8dbe19714567814eb3efd7e08f35faf49/src/components/date-picker/DatePicker.vue#L127

Reply:
Let me know if this doesn't work, and we'll look for a better or simpler solution.

Reply:
Thank you Severin.
I used a const authorizationDateValid to track the authorization date form validity. It works.